README.md

```diff
@@ -1,30 +1,49 @@
 # YourProjectName
 
-TODO documentation
+## Project Purpose
+
+This is a template repository, intended to accelerate teams creating packages for publishing on NPM.
+
+This repository is not specifically frontend focused.
+
+## Key Features
+
+The following are all handled for you and do not need to be configured:
+
+- Licensing
+- Linting
+- Formatting
+- Testing
+- Building
+- Bundling
+- Automated Versioning
+- Publishing
 
 ## Getting Started
 
-Find and replace all instances of the symbols (case-insensitive):
+Find and replace all instances of the symbols (case-insensitive) with your project name:
 
 - `your-project-name`
 - `YourProjectName`
 
-and replace it with the name of your repo.
+In order to publish to NPM you will need to add an NPM token as a GitHub secret with the name `NPM_TOKEN`.
 
-## Customizing the build
+Once you are all setup, we recommend reviewing and rewriting this README as necessary to make it more specific to your project.
+
+The last thing to do is actually start writing your code! We recommend starting from `src/index.ts` and `src/index.test.ts`.
 
-### Vite
+## Customizing the build
 
-See here for docs on how to add multiple entry points: [Vite Library Mode](https://vitejs.dev/guide/build.html#library-mode)
+The build system uses Vite. It is configured to treat `./src/index.ts` as the entry point of your repository. Feel free to change any particulars in `./vite.config.ts`. In particular if you require multiple entry points, we recommend reviewing Vite's documentation on this here: [Vite Library Mode](https://vitejs.dev/guide/build.html#library-mode)
 
-### Husky
+## Husky
```

Comment:
This was meant to be under "customise" which is why I used h3. You think flatter structure is clearer?

Reply:
I do. But happy to be wrong about that :-)